Formula/s/swiftgen.rb

```diff
@@ -11,23 +11,20 @@ class Swiftgen < Formula
     sha256 cellar: :any_skip_relocation, monterey:       "3c158aecb7ced0c489ef11f6a4c15a9d1449a40ad554388a0310116f510ec316"
   end
 
-  # https://github.com/SwiftGen/SwiftGen/issues/1030
-  disable! date: "2024-02-22", because: :does_not_build
-
   depends_on xcode: ["13.3", :build]
   depends_on :macos
 
   uses_from_macos "ruby" => :build, since: :high_sierra
 
   resource("testdata") do
-    url "https://github.com/SwiftGen/SwiftGen/archive/refs/tags/6.6.2.tar.gz"
-    sha256 "73b73e32ce22554c9db44c8edf0fa0ada33b413c73e8f991eebfaac4073df3de"
+    url "https://github.com/SwiftGen/SwiftGen/archive/refs/tags/6.6.3.tar.gz"
+    sha256 "f529be194f0ffcc85a76a6770fe3578b49e7e56ba872ce1e3aaba75982b09d32"
   end
```

Comment:
can you move this (L22-L25) down to the test block after L43? Thanks!

Reply:
Like [this](https://github.com/Homebrew/homebrew-core/pull/164869/files#diff-2abae6b151d6f382ad936c8898e188e574851f58ccf7d85a79d23d3bbd406553R36-R40) @chenrui333 ?

Reply:
yeah, perfect! Thanks!